mapreduce/datadog_checks/mapreduce/mapreduce.py

```diff
@@ -194,6 +196,15 @@ def _parse_job_specific_counters(self, init_config):
 
         return job_counter
 
+    def _get_hadoop_version(self):
+        if self.is_metadata_collection_enabled():
+            try:
+                cluster_info = self._rest_request_to_json(self.rm_address, self.CLUSTER_INFO)
+                hadoop_version = cluster_info.get('clusterInfo', {}).get('hadoopVersion', '')
```

Comment:
We shouldn't probably send version if `hadoop_version` is empty string. aka `hadoopVersion` not present.

Reply:
I thouth set_metadata threw an exception in that case but seems not. Good catch!